frontend/src/layouts/Main/Layout.tsx

```diff
@@ -0,0 +1,31 @@
+import Footer from "layouts/Footer/Footer";
+import NavigationBar from "layouts/NavigationBar/NavigationBar";
+import {
+  authenticatedNavItems,
+  unAuthenticatedNavItems,
+  settings
+} from "layouts/NavigationBar/navigation-data";
+import React, { type ReactElement } from "react";
+import { Outlet } from "react-router-dom";
+import "./Layout.css";
+import { useAuth } from "context/AuthContext";
+
+const Layout = (): ReactElement => {
+  const { isAuthenticated } = useAuth();
+  return (
+    <div className="app-wrapper">
+      <NavigationBar
+        navigationItems={
+          isAuthenticated ? authenticatedNavItems : unAuthenticatedNavItems
+        }
```

Comment:
In the unauthenticated case the right corner of the navbar should be nothing more or less than (for mobile) vertically placed "Login" and "Register" , at least according to Figma.
The current navbar implementation is for the authenticated case. settings is the menu that will open for the current logged-in user.